OPAL/tac/src/main/scala/org/opalj/tac/fpcf/analyses/cg/AndroidICCAnalysis.scala

```diff
@@ -5,10 +5,7 @@ package fpcf
 package analyses
 package cg
 
-import org.opalj.br.ClassFile
-import org.opalj.br.Method
-import org.opalj.br.ObjectType
-import org.opalj.br.ObjectVariableInfo
+import org.opalj.br.{ClassFile, Method, ObjectType, ObjectVariableInfo, PCAndInstruction}
```

Comment:
Please avoid these multi-imports. You should configure your IDE to not create them.